FFXIVClientStructs/FFXIV/Client/Game/Object/Treasure.cs

```diff
@@ -6,12 +6,93 @@ namespace FFXIVClientStructs.FFXIV.Client.Game.Object;
 [Inherits<GameObject>]
 [StructLayout(LayoutKind.Explicit, Size = 0x210)]
 public unsafe partial struct Treasure {
+
+    /// <remarks>
+    /// Has a lot of overlap with <see cref="Flags"/>.
+    /// </remarks>
+    [FieldOffset(0x1A0)] public TreasureState State;
+
+    /// <remarks>
+    /// Starts counting down from <see cref="CountdownStartingValue_Sec"/> once spawned.  Unsure what hitting zero actually does
+    /// in the unopened state.  If the treasure contains rollable items, opening the it resets this to the claim period length
+    /// and it starts counting down again (except when lootmaster is active, in which case it resets to zero).  This value is
+    /// rounded to nearest whole number and shown as the loot window timer.
+    /// </remarks>
+    [FieldOffset(0x1A4)] public float CountdownTimer_Sec;
+
+    /// <summary>
+    /// The starting value of <see cref="CountdownTimer_Sec"/> at initial object spawn.
+    /// </summary>
+    [FieldOffset(0x1A8)] public float CountdownStartingValue_Sec;
+
+    /// <summary>
+    /// The time after opening to be able to roll on or assign drops before auto-disposition of loot.
+    /// </summary>
+    [FieldOffset(0x1AC)] public float ClaimPeriodLength_Sec;
+
+    /// <summary>
+    /// The Item sheet row IDs for the contents of this treasure that have been added to the loot window.
+    /// </summary>
+    [FieldOffset(0x1B0), FixedSizeArray] internal FixedSizeArray16<uint> _lootableItemIds; // Assuming 16 long due to the location of the count field and the layout of the loot window.  Was only able to confirm up to 12.
+    
+    /// <summary>
+    /// The number of valid item IDs in <see cref="LootableItemIds"/>.
+    /// </summary>
+    [FieldOffset(0x1F0)] public byte LootableItemCount;
```

Comment:
```suggestion
    [FieldOffset(0x1F0)] public byte ItemCount;
```

Reply:
If you look at all the other places where this is done, it is the same as what I have in my requested change.